scripts/wait-for-all-codebuild.ts

```diff
@@ -29,8 +29,13 @@ const main = async () => {
   const expectedSourceVersion = process.argv[2];
   const jobsDependedOnFilepath = process.argv[3];
   const codeBuildProjectName = process.argv[4];
-  const jobsDependedOnRaw = fs.readFileSync(jobsDependedOnFilepath, 'utf8');
-  const jobsDependedOn = JSON.parse(jobsDependedOnRaw);
+  let jobsDependedOn: string[];
+  try {
+    const jobsDependedOnRaw = fs.readFileSync(jobsDependedOnFilepath, 'utf8');
+    jobsDependedOn = JSON.parse(jobsDependedOnRaw);
+  } catch {
+    jobsDependedOn = [jobsDependedOnFilepath];
```

Comment:
in what cases would this condition hit?

Reply:
If jobsDependedOnFilepath is the name of a job, a file is not needed. We take jobsDependedOnFilepath and create an array with it as a single element. 
For example, this allows us to call:
_waitForJobs <jobname>
or
_waitForJobs <filepath with job names list>

Reply:
We should make a backlog item to revisit this. There's a lot of implicit nuance here like it seems that `jobsDependedOnFilepath` is not always a file path or what happens if the JSON content is malformed such that `JSON.parse()` fails but was intended to be parsed?